source/modules/MEI.shared.xml

```diff
@@ -1894,7 +1894,22 @@
         <defaultVal>5.0-dev</defaultVal>
         <valList type="closed">
           <valItem ident="5.0-dev">
-            <desc xml:lang="en">Development version of MEI 5.0</desc>
+            <desc xml:lang="en">Development version of MEI 5.0.</desc>
+          </valItem>
+          <valItem ident="5.0-dev+anyStart">
```

Comment:
This doesn't work, because `anyStart` isn't compatible with any other schema. See following example.

```xml
<editorialDecl xmlns="http://www.music-encoding.org/ns/mei" meiversion="5.0-dev+anyStart">
  <p>A test to make a point.</p>
</editorialDecl>
```

This validates with `anyStart` schema, but not with `all`.

Reply:
good point

Reply:
Nevertheless, it would work for all files that have `meiversion="5.0+anyStart` and start with one of the allowed start elements of mei-all…

Reply:
What do you think, should anyStart then also disallow any other value for `@meiversion`?

Reply:
I think 5.0+anyStart can safely be available in mei-all, because  `@meiversion` is only allowed on the start-element in mei-all and thus would be valid against both schemata.

Reply:
> ```xml
><editorialDecl xmlns="http://www.music-encoding.org/ns/mei" meiversion="5.0-dev+anyStart">
>  <p>A test to make a point.</p>
></editorialDecl>```
>This validates with anyStart schema, but not with all.

This is absolutely okay because the error reported is not solely due to `@meiversion` but because `editorialDecl` is not a valid start element in mei-all. Disallowing `meiversion="5.0"` in anyStart would make more sense, or adding a schematron rule that reports if the root element of an anyStart-document is not one that is also allowed in `5.0` and `meiversion` is not anyStart.